### PR TITLE
Task 7: working with classes (variant 3, Point and Line)

### PR DESCRIPTION
**Pull request by Pasichnichenkoilya**

### CODE REVIEW

File: lab7/tests/unit/test_line.py [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+        x = (self.point_a.x + self.point_b.x) / 2
+        y = (self.point_a.y + self.point_b.y) / 2
+
+        return Point(x, y)
+
+    def get_length_projection_x(self):
+        return abs(self.point_b.x - self.point_a.x)
+
+    def get_length_projection_y(self):
+        return abs(self.point_b.y - self.point_a.y)
```
Comment (Pasichnichenkoilya) [inline lab7/tests/unit/test_line.py:35]: How do I fix this??

File: lab7/tests/unit/test_point.py [hunk 2]
```diff
@@ -1,4 +1,10 @@
-from lab7.point import Point
+class Point:
+    def __init__(self, x: float, y: float):
+        self.x = x
+        self.y = y
+
+    def __str__(self):
+        return f'Point(x: {self.x}, y: {self.y})'
```
Comment (Pasichnichenkoilya) [inline lab7/tests/unit/test_point.py:7]: And this...

File: lab7/tests/unit/test_line.py (hunk 1 above)
Comment (vloooo) [reply lab7/tests/unit/test_line.py:35]: you don't need to redefine your class in a test file. Just import it

File: lab7/line.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import math as m
+from point import Point
+
+
+class Line:
+    def __init__(self, point_a: Point, point_b: Point):
+        self.point_a = point_a
```
Comment (vloooo) [inline lab7/line.py:7]: please add access modifiers

File: lab7/tests/unit/test_line.py (hunk 1 above)
Comment (Pasichnichenkoilya) [reply lab7/tests/unit/test_line.py:35]: It says "No module named 'line'" when i try to do it

Comment (vloooo) [reply lab7/tests/unit/test_line.py:35]: you should mention a whole path from the root folder

Comment (vloooo) [reply lab7/tests/unit/test_line.py:35]: just wait a bit, tomorrow we'll consider it

Comment (Pasichnichenkoilya) [reply lab7/tests/unit/test_line.py:35]: ok

**Review by vloooo — APPROVED** [review]

**Pull request merged**